cypress/e2e/smoke/landing.cy.js

```diff
@@ -0,0 +1,34 @@
+describe('Landing page', () => {
+  it('redirects to welcome page', () => {
+    cy.visit('/')
+
+    cy.url().should('include', '/welcome')
+  })
+
+  it('features discoverability cookies', () => {
+    cy.visit('/')
+
+    let preferencesModal = cy.contains("We use cookies to provide").parent()
```

Comment:
Let's add the same data-testid="cookies-banner-form"  for the "class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 styles_container__12f0T css-c28ola" and reuse the full "landing.spec.js" test from the safe-react with testId instead of .parent()

Reply:
I would only advise we not `data-testId`s unless we really have to, e.g. if we were to alter the content of the banner, the tests won't break.

Reply:
Opted to use `cy.contains('We use cookies to provide').parent('div')` to find the Cookies banner parent element.